README.md

```diff
@@ -132,22 +132,22 @@ schema:
 `schema.edge` is a **required** parameter that describes the schema information of the inserted edge. Each edge contains the following three properties:
 
 * `name`: The edge's name.
-* `withRanking`: Specifies the `rank` value of the given edge, used to tell different edges share the same edge type and vertices.
```

Comment:
`withRanking`: Specifies the `rank` value of the given edge. It is used to tell different edges sharing the same edge type and vertices.